application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue

```diff
@@ -251,6 +254,9 @@ export default {
       if (!newMessage || !newMessage.trim()) {
         return;
       }
+      if (newMessage.includes('<img')) {
```

Comment:
IMO you should search all strings occurrences with regex <img src="*" > inside newMessage and replace them after injecting the alt.
this should be done only if the search returns occurrences.